lona/html/data_binding/inputs.py

```diff
@@ -74,12 +79,31 @@ def disabled(self):
 
     @disabled.setter
     def disabled(self, new_value):
+        if not isinstance(new_value, bool):
+            raise ValueError('disabled is a boolean property')
```

Comment:
This doesn't allow to use string html
```py
HTML('<input type="checkbox" disabled/>')
```

Reply:
I haven't created an issue yet. But seems it's good time to say that.

I think it's better to use `node.attributes` when parsing html string instead of passing `kwargs` to constructor here:
https://github.com/lona-web-org/lona/blob/049201b78772ede3a83590ae54958872fb1c4b49/lona/html/parsing.py#L129

Reply:
For example following `HTML('<div nodes="aaa">bbb</div>')` produces `Div("aaa", "bbb")`, but I expect to get `Div("bbb", attributes={"nodes": "aaa"})`.
I expect that string will be treated as html. Of course it's ok if lona will add some attributes like `data-lona-node-id` and `data-lona-events`.

Reply:
Oh! You are right! I liked that the high level properties get used to set boolean attributes, but you would have to make many exceptions to make this work. Ok we need to use ``Node.attributes`` instead of ``Node.__init__`` to set data.

Should we open an dedicated issue for this or should i append changes for that here?

Reply:
I don't mind to see changes in this PR.

Reply:
@maratori fixed! Thanks for bringing this up

Reply:
Here will be `text` always because `attributes` now contains only `tag_name` and `self_closing_tag`:
https://github.com/lona-web-org/lona/blob/808c42c96a5faaa6ef11ee2ce6a0c4c4df2545a1/lona/html/parsing.py#L94-L95